packages/actor-function-factory-expression-function-bnode/lib/ActorFunctionFactoryExpressionFunctionBnode.ts

```diff
@@ -0,0 +1,33 @@
+import type {
+  IActionFunctionFactory,
+  IActorFunctionFactoryArgs,
+  IActorFunctionFactoryOutput,
+  IActorFunctionFactoryOutputTerm,
+} from '@comunica/bus-function-factory';
+import {
+  ActorFunctionFactory,
+} from '@comunica/bus-function-factory';
+import type { IActorTest } from '@comunica/core';
+import { SparqlOperator } from '@comunica/expression-evaluator';
+import { ExpressionFunctionBnode } from './ExpressionFunctionBnode';
+
+/**
+ * A comunica ExpressionFunctionBnode Function Factory Actor.
+ */
+export class ActorFunctionFactoryExpressionFunctionBnode extends ActorFunctionFactory {
+  public constructor(args: IActorFunctionFactoryArgs) {
+    super(args);
+  }
+
+  public async test(action: IActionFunctionFactory): Promise<IActorTest> {
+    if (action.functionName === SparqlOperator.BNODE || action.requireTermExpression) {
```

Comment:
Is this correct? Should this become an `&&`?

Reply:
You're right! Wow, logic escaped me last night! It should be `action.functionName === SparqlOperator.BNODE && !action.requireTermExpression`